<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manageUser</class>
 <widget class="QMainWindow" name="manageUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理录入信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: white;
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}
QPushButton{
	background-color: skyblue;
}
QLabel{
	font: 12pt &quot;YaHei Monaco Hybird&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>640</y>
      <width>300</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="return_bt">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="user_tb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>451</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>已录入信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
